i18n/en.yaml

```diff
@@ -73,7 +83,7 @@ alias_new_email_c_text_2: >
 alias_new_email_c_text_3: 50+ and counting businesses trust SimpleLogin to run their emails.
 
 # Testimonials Section
-testimonial_title: SimpleLogin is loved by users
+testimonial_title: Email aliasing loved by users
```

Comment:
what do you think about "People love Email Aliases"? Email aliasing sounds a bit weird.